empire

```diff
@@ -1500,8 +1500,12 @@ if __name__ == '__main__':
     if args.version:
         print(empire.VERSION)
 
-    if args.reset_empire:
-        subprocess.call("./setup/reset.sh")
+    if args.reset:
+        choice = input("\n [>] Would you like to reset your Empire instance? [Y or N]: ")
```

Comment:
```suggestion
        choice = input("\n [>] Would you like to reset your Empire instance? [y/N]: ")
```

Assuming "n" input just skips this, the convention is to capitalize whichever is the default.